package.json

```diff
@@ -29,9 +29,10 @@
     "homepage": "https://github.com/ArrowDFMs/arrow-markets-sdk#readme",
     "dependencies": {
         "axios": "0.24.0",
+        "dayjs": "^1.11.5",
```

Comment:
Can we fix the version of dayjs and dayjs-plugin-utc ? Many times, the `^` in front of the version will cause issues down the line.